README.md

```diff
@@ -1,60 +1,58 @@
-NI Linux Real-Time Distribution
-======
+# NI Linux Real-Time
 
-Introduction
-------
+## Introduction
 
-Downloads the NI Linux Real-Time (NILRT) source code and
-provides scripts to aid in building images and packages.
+This project builds packages and images for the NI Linux RT distribution.
 
-Build Steps
-------
+NI Linux RT is a
+[Real-Time-scheduling](https://rt.wiki.kernel.org/index.php/CONFIG_PREEMPT_RT_Patch)
+enabled OS, for use on NI's embedded hardware devices. This project uses the
+[OpenEmbedded](http://www.openembedded.org/wiki/Main_Page) framework to build
+and package the kernel, software, images, and installation media which compose
+NI Linux RT.
 
-1. Get the source by running the following commands:
-
-        git clone https://github.com/ni/nilrt.git
-        cd nilrt
-        git checkout nilrt/<release>
-        git submodule init
-        git submodule update --remote --checkout
-
-2. Set up your shell environment to run bitbake:
-
-    Source NILRT's environment script by running the following command:
-
-        . ni-oe-init-build-env
-
-    This will setup the needed environment variables and build
-configuration files for building through OpenEmbedded build system.
-Note that the configuration files (that exist in the build/conf
-directory) include some basic default configurations, allowing
-modification or overriding of these default configurations.
+Community (non-NI) developers can use this project to build packages for
+open-source software projects, which are installable to their existing NI Linux
+RT devices - including custom Linux kernels and kernel modules.
 
-    Set an appropriate MACHINE variable so that bitbake can tune builds
-of NILRT for your hardware.
 
-    Run the following command to configure for ARM targets:
-
-        export MACHINE=xilinx-zynqhf
-
-    or the following command to configure for x64 targets:
-
-        export MACHINE=x64
-
-    **NOTE** It's not recommended to run bitbake for different
-MACHINE's in the same workspace (build directory).
-
-3. Build the package or packages that you want for your target.
-For example, to build Python, Ruby, and Apache for Zynq targets, run the
-following commands:
+## Building Packages
```

Comment:
Setting up the docker environment should probably be before this section? Or atleast a note about the environment needed to build packages.

Reply:
I've added a paragraph in step 2 which recommends using the Dockerfile to create a build container, and I deleted the final section entirely. There's a lot more that a dev would need to know about docker containers to get one to work as their bitbake environment (like how to bind mount the workspace, passthrough host networking, et c.) than the `docker build` command that was there. And I don't think the README is the appropriate place to do a whole walkthrough on using docker containers.

Maybe we can add a doc like that in the future.